### PR TITLE
chore: add agile questions

### PR DESCRIPTION
**Pull request by RitaDee**
## Summary of changes
Add questions on Agile.

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/agile-quiz.ts [hunk 1]
```diff
@@ -438,7 +438,27 @@ const agileQuiz = [
     Explanation:
       "Working software is the primary measure of progress in Agile, according to the seventh principle of the Agile Manifesto. This means that Agile teams focus on delivering software that meets the customer’s needs and provides value. By delivering working software frequently, Agile teams can demonstrate their progress and achievements",
     Link: "https://www.freecodecamp.org/news/you-say-your-team-is-agile-but-that-word-may-not-mean-what-you-think-6dd26eaf9b21/"
-  }
+  },
+  {
+    Question: "In agile, why is it generally advisable to avoid assigning a very high number of points (e.g 13 points) to a single user story?",
+    Answer: "Because high point stories cannot be completed in a single sprint.",
+    Distractor1: "Because high point stories are reserved for critical features only.",
+    Distractor2: "Because high point stories indicate a lack detail in the user story.",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:446]:
```suggestion
    Distractor2: "Because high point stories indicate a lack of detail in the user story.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/agile-quiz.ts (hunk 1 above)
Comment (RitaDee) [reply src/data/agile-quiz.ts]: Noted with thanks.
I have fixed it.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**